### ISSUE TITLE
go install github.com/minio/mc@latest](http://github.com/minio/mc@latest broken

### ISSUE THREAD

**Issue by k4leung4** [minio/mc#4465]
## Expected behavior
minio is installed

## Actual behavior
installation fails with 

```
go: downloading github.com/minio/mc v0.0.0-20230202165714-2679713ec403
go: github.com/minio/mc@latest (in github.com/minio/mc@v0.0.0-20230202165714-2679713ec403):
	The go.mod file for the module providing named packages contains one or
	more replace directives. It must not contain directives that would cause
	it to be interpreted differently than if it were the main module.
```

## Steps to reproduce the behavior

`go install github.com/minio/mc@latest](http://github.com/minio/mc@latest`

```
go version
go version go1.19.5 darwin/arm64
```

`go install github.com/minio/mc@6e97f161c85b41548ae452e0681799fa072e5ffc` works fine

## mc --version
- (paste output of `mc --version`)

**Comment by k4leung4** [comment]
issue acknowledge here
https://github.com/minio/mc/pull/4464#issuecomment-1414190887

workaround is to use `make`

**Comment by harshavardhana** [comment]
fixed via 3aebb3362b18225180daa9beb92685d73110b47c
